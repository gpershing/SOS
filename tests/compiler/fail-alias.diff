diff: ./tests/compiler/tests/compiler/fail-alias.err: No such file or directory
