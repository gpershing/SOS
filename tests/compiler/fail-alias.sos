alias a = int
alias a = int
