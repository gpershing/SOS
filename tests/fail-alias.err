Fatal error: exception Failure("Cannot create an alias with preexisting name int")
