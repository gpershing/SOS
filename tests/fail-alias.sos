//aliasing type id is already defined. could have problems if aliasing float as int, etc.

alias int = int
