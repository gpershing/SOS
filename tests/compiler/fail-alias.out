Cannot create an alias with preexisting name 
